### ISSUE TITLE
Touch gesture swipe on the `BitCarousel` component is not working on mobile

### ISSUE THREAD

**Issue by msynk** [bitfoundation/bitplatform#2804]
after recent development and improvement of the BitCarousel, the swipe feature is still not working.

**Comment by msynk** [comment]
the issue is coming from the preventDefault of the `touchmove` event. there are two bugs/problems with the current implementations in the Blazor.

- the first one is discussed here: https://github.com/dotnet/aspnetcore/issues/18449 
which can be simply bypassed by setting a dummy handler to the `@ontouchmove` event on any element 😳😲🤦‍♂️

- the second problem is a little more complicated and related to the `passive` property of this particular event.
more info here: https://chromestatus.com/feature/5093566007214080 and https://developer.chrome.com/blog/scrolling-intervention/
basically, in order to have a better browsing experience, the chromium decided to apply `passive:true` for some critical events such as `touchmove` on mobile versions, which makes the calling to preventDefault ignored. so in order to fix this issue, the handler needs to be set with this property set to false.
like this for example: `el.addEventListener('touchmove', handler, { passive: false })`
